server_src/runners/state.js

```diff
@@ -3409,29 +3416,21 @@ class Runner {
       if (settings.systemCommands.serverRestart !== "") {
         farmPrinters[index].settingsServer.commands.serverRestartCommand =
           settings.systemCommands.serverRestart;
-      } else {
-        settings.systemCommands.serverRestart =
-          farmPrinters[index].settingsServer.commands.serverRestartCommand;
       }
       if (settings.systemCommands.systemRestart !== "") {
         farmPrinters[index].settingsServer.commands.systemRestartCommand =
           settings.systemCommands.systemRestart;
-      } else {
-        settings.systemCommands.systemRestart =
-          farmPrinters[index].settingsServer.commands.systemRestartCommand;
       }
       if (settings.systemCommands.systemShutdown !== "") {
         farmPrinters[index].settingsServer.commands.systemShutdownCommand =
           settings.systemCommands.systemShutdown;
-      } else {
-        settings.systemCommands.systemShutdown =
-          farmPrinters[index].settingsServer.commands.systemShutdownCommand;
```

Comment:
legacy code from an even older update method, these we're all pointless as I'm re-updating what was sent.